_inc/client/components/jumpstart/index.jsx

```diff
@@ -23,25 +26,40 @@ class JumpStart extends React.Component {
 	static displayName = 'JumpStart';
 
 	activateButton = () => {
-		return <Button
-			primary={ true }
-			onClick={ this.props.jumpStartActivate }
-			disabled={ this.props.isJumpstarting }
-		>
-			{ this.props.isJumpstarting ? __( 'Activating recommended features…' ) : __( 'Activate recommended features' ) }
-		</Button>;
+		return (
+			<div>
```

Comment:
The fact that this `<div> ` is rendered inside a `<p>` is generating these warnings:

```
warning.js:33 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>. See JumpStart > p > ... > div.
... > p.
```

Reply:
Thank you. Fixed it by removing the additional `p`, as you suggested. It does not change the look of things.